tern/formats/default/generator.py

```diff
@@ -18,6 +18,9 @@
 # global logger
 logger = logging.getLogger(constants.logger_name)
 
+# Licenses not to be included in the final report
+ignore_list = ['UNKNOWN', 'unknown', 'Unknown']
```

Comment:
I would rather leave anything reported as "unknown" even if it is added by default.

Reply:
Ok, this means that `UNKNOWN` will show up in the list of licenses in the default report.

Reply:
PR and commit message is updated :)